examples/sys_sysmon/sysmon-client.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import base64
```

Comment:
There should be 2 empty lines between each function definition.

Reply:
done